tsconfig.json

```diff
@@ -3,14 +3,18 @@
         "target": "es5",
         "skipLibCheck": true,
         "strict": true,
-        "lib": ["es5", "dom"],
+        "lib": [
+            "es5",
+            "dom"
+        ],
         "types": [
             "cypress",
             "node"
         ]
     },
     "include": [
-        "cypress/**/*.ts"
+        "cypress/**/*.ts",
+        "cypress/page/index.js"
```

Comment:
since you already have the `"cypress/**/*.ts"` pattern, the index.ts file will now be taken by the config, you won't need `"cypress/page/index.js"`

Reply:
did you push the changes? I don't see them